packages/itinerary-body/src/styled.tsx

```diff
@@ -349,7 +332,7 @@ export const LegIconContainer = styled.span`
   img,
   svg {
     margin-right: 6px;
-    max-height: 24px;
+    height: 24px;
```

Comment:
From the browser dev tools, this will not achieve the desired effect.
<img width="378" alt="image of Chrome dev tools with grayed out image width and height props, and popup stating that the display:inline property prevents the size from taking effect." src="https://user-images.githubusercontent.com/56846598/228891373-25ffc39d-845f-411b-84df-78a02e0d135e.png">

Reply:
Does this apply to `img` as well or just `svg`?

Reply:
Just SVG I think. Unless you want to make the SVG `display:inline-block`.

Reply:
Yeah I've found those warnings to be very buggy especially for more complex css